api/spec/v1.yaml

```diff
@@ -2076,12 +2076,12 @@ components:
             and the constructor parameters. When run, this code generates the runtime bytecode.
             Can be omitted for contracts that were created by another contract, as opposed 
             to a direct `Create` call.
-        # runtime_bytecode:
-        #   type: string
-        #   format: byte
-        #   description: |
-        #     The runtime bytecode of the smart contract. This is the code stored on-chain that
-        #     descibes a smart contract.
+        runtime_bytecode:
+          type: string
+          format: byte
+          description: |
+            The runtime bytecode of the smart contract. This is the code stored on-chain that
+            descibes a smart contract. This field is normally present, except for contracts
```

Comment:
Missing line?

Also, since the runtime bytecode analyzer might be behind/stalled, the field will be missing for contract accounts that haven't been processed yet. Do you think it's worth checking the `evm_contract_analysis` table and providing something akin to `pending` if `is_contract == NULL`? Or perhaps amending the description is sufficient in this case